workflow-controller/controller-service/src/main/java/io/hyscale/controller/commands/HyscaleGetServiceCommand.java

```diff
@@ -42,7 +44,8 @@
 	 * Provides usage of this command to the user
 	 */
 	@Override
-	public void run() {
-		new CommandLine(new HyscaleGetServiceCommand()).usage(System.out);
+    public Integer call() throws Exception {
+	    new CommandLine(new HyscaleGetServiceCommand()).usage(System.out);
+	    return 0;
```

Comment:
Input validation failure right ?

Reply:
It covers the case where command itself is not complete, rather than input validation. 
We can have a separate exit code for such cases, or should I set it as invalid input for the time being?